How-To-Host-Feed.md

```diff
@@ -332,11 +332,11 @@ See https://help.sonatype.com/repomanager3/high-availability-introduction for de
 #### ProGet
 
 * Windows Server
-* CPU - 4 cores (Recommend more)
-* RAM - 8GB+
-* Storage (HDD) - We recommend 2-5TB of free space for a local file store of artifacts (default install needs 20GB).
-* Database: SQL Server. Assume high specs for SQL Server, pretty much the highest specs you can configure on a Windows machine. 16+ cores (CPU), 64GB+ RAM, 5TB+, etc
-* Chocolatey Repository Type: Chocolatey or NuGet
+* CPU - 2 cores (4+ cores recommended for large instances)
+* RAM - 4GB+ (8GB recommended for large instances)
+* Storage (HDD) - We recommend 2-5TB of free space for a local file store of artifacts (default install needs 10GB).
+* Database: Higher specs may be required if creating more than a simple feed repository. Minimum, SQL Server Express 2012, (install available during download) 1GB RAM, 6GB disk space.
```

Comment:
We are in the commercial section - organizations that are going to use these things are not concerned with Minimums and we are not talking minimums. We are talking Chocolatey's recommendations for folks to have a good time. Please leave this as it was.